internal/proxy/proxy.go

```diff
@@ -546,6 +550,11 @@ func NewClient(ctx context.Context, d cloudsql.Dialer, l cloudsql.Logger, conf *
 	for _, inst := range conf.Instances {
 		m, err := c.newSocketMount(ctx, conf, pc, inst)
 		if err != nil {
+			if conf.IgnoreFailedConnections {
```

Comment:
A test for this block of code would be good too.

Reply:
Done